stock_release_channel_cutoff/models/stock_release_channel.py

```diff
@@ -17,23 +20,20 @@ class StockReleaseChannel(models.Model):
     # Technical field for warning on kanban view
     cutoff_warning = fields.Boolean(compute="_compute_cutoff_warning")
 
-    @api.depends_context("tz")
-    @api.depends("cutoff_time", "state")
+    @api.depends("cutoff_time", "state", "process_end_time_tz")
```

Comment:
Why is the prcess_end_date not in the depends?
```suggestion
    @api.depends("cutoff_time", "state", "process_end_date", "process_end_time_tz")
```
or should it be
```
    @api.depends("cutoff_time", "state", "process_end_time", "process_end_time_tz")
```
since cutoff_warning is not stored it should be good to use the none stored field process_end_date

Reply:
added